### PR TITLE
use 2 characters

### PR DESCRIPTION
**Pull request by terite**
except when using "WenQuanYi Zen Hei Sharp", because then the text is too big.

depends on #1

english
chinese

### CODE REVIEW

**Comment by bradyoo** [comment]
LGTM

File: routes/index.js [hunk 1]
```diff
@@ -41,16 +47,20 @@ router.get('/', function(req, res, next) {
...
   }
 
-  ctx.font = Math.ceil(size * 0.8) + 'px ' + fonts;
-  var te = ctx.measureText(initial);
-  ctx.fillText(initial, (size - te.width) * 0.5, size * 0.8 + pad);
+  ctx.font = Math.ceil(size * 0.5) + 'px ' + fonts;
+  var textSize = ctx.measureText(text);
+
+  var xPos = (size - textSize.width) * 0.5;
+  var yPos = size * 0.667;
```
Comment (bradyoo) [inline routes/index.js]: You might need to do something fancy to check if the the text is actually 2 characters long.

Comment (terite) [inline routes/index.js]: They look centered to me
d
de

**Comment by bradyoo** [comment]
LGTM

**Pull request merged**